src/main/python/ayab/engine/engine_fsm.py

```diff
@@ -56,13 +63,14 @@ class StateMachine(object):
     @date   June 2020
     """
 
-    def set_transitions(self, parent):
+    def set_transitions(self, parent:Engine)->None:
         """Define transitions between states for Finite State Machine"""
 
         # Events that trigger state changes
-        self.CONNECT.addTransition(parent.port_opener, self.VERSION_CHECK)
+        self.CONNECT.addTransition(parent.port_opener, self.VERSION_CHECK) #type: ignore
 
-    def _API6_connect(control, operation):
+    @staticmethod
+    def _API6_connect(control:Control, operation:Operation)->Output:
```

Comment:
these are only ever called in a dynamic invocation `getattr` way- they're being used as static methods so we might as well be explicit about that.